src/covid19sim/interventions.py

```diff
@@ -734,3 +735,51 @@ def revert_city(self, city):
             city ([type]): [description]
         """
         pass
+
+def get_intervention(key, RISK_MODEL=None, TRACING_ORDER=None, TRACE_SYMPTOMS=None, TRACE_RISK_UPDATE=None, SHOULD_MODIFY_BEHAVIOR=True):
+    """
+    [summary]
+
+    Args:
+        key ([type]): [description]
+        RISK_MODEL ([type], optional): [description]. Defaults to None.
+        TRACING_ORDER ([type], optional): [description]. Defaults to None.
+        TRACE_SYMPTOMS ([type], optional): [description]. Defaults to None.
+        TRACE_RISK_UPDATE ([type], optional): [description]. Defaults to None.
+        SHOULD_MODIFY_BEHAVIOR (bool, optional): [description]. Defaults to True.
+
+    Raises:
+        NotImplementedError: [description]
+
+    Returns:
+        [type]: [description]
+    """
+    if key == "Lockdown":
+        return Lockdown()
+    elif key == "WearMask":
+        return WearMask(MASKS_SUPPLY)
+    elif key == "SocialDistancing":
+        return SocialDistancing()
+    elif key == "Quarantine":
+        return Quarantine()
+    elif key == "Tracing":
```

Comment:
`Tracing` is a class in this file. I don't think there is a global variable with that name. I think if it's giving error, it might be due to other reasons.